db/migrate/20170822104447_add_katello_content_to_image.rb

```diff
@@ -1,5 +0,0 @@
-class AddKatelloContentToImage < ActiveRecord::Migration[4.2]
-  def change
-    add_column :docker_container_wizard_states_images, :katello_content, :text
```

Comment:
should this column be dropped?

Reply:
Yes.  Drop . These point to purely foreman docker entities.